packages/framework/src/Component/UploadedFile/UploadedFileFactory.php

```diff
@@ -2,8 +2,24 @@
 
 namespace Shopsys\FrameworkBundle\Component\UploadedFile;
 
+use Shopsys\FrameworkBundle\Component\FileUpload\FileUpload;
+use Shopsys\FrameworkBundle\Component\UploadedFile\Config\UploadedFileEntityConfig;
+
 class UploadedFileFactory implements UploadedFileFactoryInterface
 {
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\FileUpload\FileUpload
+     */
+    private $fileUpload;
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\FileUpload\FileUpload $fileUpload
+     */
+    public function __construct(FileUpload $fileUpload)
```

Comment:
should be mentioned in the docs as well...and the `UploadedFileFactoryInterface` should contain the constructor now, right?